### DISCUSSION TITLE
Cannot install timeout on sigs.securityonion.net

### DISCUSSION THREAD

**Discussion by carolsworld** [Security-Onion-Solutions/securityonion#12178] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

24

### Storage for /

how to check

### Storage for /nsm

how to check

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I tried several times but unable to install security onion 2.4 (with the latest iso securityonion-2.4.30-20231228.iso ). Except that i am using VirtualBox, I followed every instruction on the videos on your YouTube channel. Every time there is a message related to timeout and try 5 times to access or use something related to sigs.securityonion.net. then, there are a lot of [info] related things prompts up. I have no idea what is this. Is it about firewall settings. If yes, I wonder if there is tutorial showing first-time users on how to make sure the firewall and network settings are done properly? 

To assist the case understanding, please refer to the picture below. When the installation and configuration finished, it shows the blue screen showing install has a problem. I am not sure how the /root/errors/log could be extracted.

![1](https://github.com/Security-Onion-Solutions/securityonion/assets/43054364/42a6e783-dcbe-43db-989e-0a562e3eedae)
![2](https://github.com/Security-Onion-Solutions/securityonion/assets/43054364/1f879671-f77b-4327-b445-4fb227ab8f4e)
![3](https://github.com/Security-Onion-Solutions/securityonion/assets/43054364/ed680565-624d-4cd4-8057-83de7196ffc8)
![7](https://github.com/Security-Onion-Solutions/securityonion/assets/43054364/0f837e49-9736-4b6c-b9d9-e2d93f9dc67e)
![Settings](https://github.com/Security-Onion-Solutions/securityonion/assets/43054364/4d6f90fb-ded1-49c7-86c2-0840900d5135)
![VMsetting](https://github.com/Security-Onion-Solutions/securityonion/assets/43054364/6f94ca2d-0a0c-4fba-8a40-68e062848346)
![VMsetting2](https://github.com/Security-Onion-Solutions/securityonion/assets/43054364/a6f8b7a8-891b-46f9-8ee8-dd0b0ff0c325)
![VMsetting3](https://github.com/Security-Onion-Solutions/securityonion/assets/43054364/7ce75b8b-82b7-4c0c-9381-a57d641fa9bf)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment]
My guess is that you're trying to do a standard install (which will pull some things from the internet), but that your virtual box NAT / network settings are misconfigured. So the VM doesn't have internet access.

You could likely resolve this by doing an airgap install on the VM so it uses resources only pulled from the ISO and does not require internet access https://docs.securityonion.net/en/2.4/airgap.html

Review your virtual box network configuration it looks like you have your ```NatNetwork``` set to ```10.0.2.0/24``` but then the virtual machine running Security Onion has an IP address in the network  ```192.168.64.0/24```.
